src/components/themed/TransactionListTop.js

```diff
@@ -131,33 +131,44 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
     )
   }
 
+  /**
+   * If the parent chain supports staking, query the info server if staking is
+   * supported for this specific asset. While waiting for the query, show a
+   * spinner.
+   */
   renderStakingBox() {
     const { theme, currencyCode, stakingBalances, fiatSymbol, fiatCurrencyCode, pluginId } = this.props
     const { stakePolicies } = this.state
     const styles = getStyles(theme)
 
-    const isStakingPolicyAvailable = stakePolicies.some(stakePolicy => {
-      return [...stakePolicy.rewardAssets, ...stakePolicy.stakeAssets].some(asset => asset.pluginId === pluginId && asset.currencyCode === currencyCode)
-    })
+    const isStakingPolicyAvailable =
+      stakePolicies != null &&
+      stakePolicies.some(stakePolicy => {
+        return [...stakePolicy.rewardAssets, ...stakePolicy.stakeAssets].some(asset => asset.pluginId === pluginId && asset.currencyCode === currencyCode)
+      })
```

Comment:
I think it would look clearer if we check `this.state.stakePolicies.length === 0` instead of allowing `this.state.stakePolicies`. This would mean we would not require the `| void` extension to the type.